documentation/modules/deploying/proc-using-access-operator.adoc

```diff
@@ -0,0 +1,130 @@
+
+.Procedure
+
+. Configure a `KafkaAccess` custom resource to bind your applications to the Kafka cluster. 
++
+** Specify the name of the `Kafka` resource to connect to the Kafka cluster.
+** (Optional) Specify the name of the listener used to connect to the Kafka cluster.
+** Specify the `KafkaUser` resources representing users requiring access to the Kafka cluster.
++
+Make sure the name, namespace and listener in the `KafkaAccess` custom resource match those of your Kafka instance.
++
+.Example access configuration
+[source,yaml]
+----
+apiVersion: access.strimzi.io/v1alpha1
+kind: KafkaAccess
+metadata:
+  name: my-kafka-access
+spec:
+  kafka:
+    name: my-cluster
+    namespace: kafka
+    listener: tls
+  user:
+    kind: KafkaUser
+    apiGroup: kafka.strimzi.io
+    name: my-user
+    namespace: kafka
+----
+
+. Create the `KafkaAccess` resource in Kubernetes:
++
+[source,shell]
+----
+kubectl apply -f <kafka_access_config_file> -n strimzi-access-operator
+----
+
+. Check the status of the `KafkaAccess` custom resource: 
++
+[source,shell]
+----
+kubectl get kafkaaccess my-kafka-access -o yaml
+----
++
+The status of the resource is updated once the operator creates the binding `Secret`. 
++
+.Status shows binding secret created 
+[source,yaml]
+----
+# ...
+status: 
+  binding: 
+    name: 
+      kafka-binding
+----
+
+. Inspect the created binding secret: 
++ 
+[source,shell]
+kubectl get secret kafka-binding -o yaml
++
+The secret contains connection details and user credentials in various formats to accommodate the requirements of different development environments and applications.
++
+.Example binding secret
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: kafka-binding
+type: servicebinding.io/kafka
+data:
+  type: kafka
+  provider: strimzi
+  # Kafka cluster connection properties # <1>
+  bootstrap.servers: my-cluster-kafka-bootstrap:9092
+  bootstrap-servers: my-cluster-kafka-bootstrap:9092  
+  bootstrapServers: my-cluster-kafka-bootstrap:9092
+  # Security protocol option # <2>
+  security.protocol: SSL
+  securityProtocol: SSL
+  # TLS enabled
+  ssl.truststore.crt: my-cluster-cluster-ca-cert # <3>
+  # SCRAM-SHA-512 authentication properties # <4>
+  username: admin
+  password: password
+  sasl.jaas.config: com.example.ScramLoginModule required;
+  sasl.mechanism: SCRAM-SHA-512
+  saslMechanism: SCRAM-SHA-512
+  # mTLS authentication properties # <5>
+  ssl.keystore.crt: user.crt
+  ssl.keystore.key: user.key
+----
+<1> Comma-separated host:port connection details for Kafka clusters.
+<2> The security protocol used for communication with Kafka, which can be `PLAINTEXT`, `SSL`, `SASL_PLAINTEXT` or `SASL_SSL`.
+<3> The Strimzi cluster CA certificate if TLS is enabled.
+<4> Credentials for SCRAM-SHA-512 client authentication.
+<5> Credentials for mTLs client authentication. 
```

Comment:
I guess 4 and 5 aren't present at the same time but which one is there depends on the type of authentication chosen within the KafkaUser, so SCRAM or mTLS. Should it be made clearer in the example?

Reply:
I've added something to the paragraph before the example:

The secret contains connection details and user credentials in various formats to accommodate the requirements of different development environments and applications. 
The user credentials included depend on the authentication method specified for the `KafkaUser` resource. 
For example, SCRAM-SHA-512 credentials are provided if that type of authentication is used.